client/v3/go.mod

```diff
@@ -4,13 +4,18 @@ go 1.15
 
 require (
 	github.com/dustin/go-humanize v1.0.0
+	github.com/golang/protobuf v1.4.3 // indirect
```

Comment:
Upgrate to protobuf 1.4.x is an additional challenge due to gogo-protobuf incompatibility. 
We need to decouple these to problems. 

go-grpc started to depend on protobof 1.4.x in version 1.34. Prior to it 
it was depending on protobuf 1.3.3 (see https://github.com/grpc/grpc-go/commit/06c094c3ab224883997e33ad8e1649a80f24ee0f) 

I recommend sticking to protobuf 1.3.3 and in first shot upgrading to grpc not newer than 1.33.